Makefile

```diff
@@ -683,6 +684,20 @@ test-capd-junit: $(SETUP_ENVTEST) $(GOTESTSUM) ## Run unit and integration tests
 	$(GOTESTSUM) --junitfile $(ARTIFACTS)/junit.infra_docker.xml --raw-command cat $(ARTIFACTS)/junit.infra_docker.stdout
 	exit $$(cat $(ARTIFACTS)/junit.infra_docker.exitcode)
 
+.PHONY: test-runtime-sdk-test-extension
```

Comment:
I assume we need those targets as the main target is only for the main module and the unit tests below test/ are not run at all today?

Is the new target scoped to test/extension or to test/ ? (I think the path should match the target name)